apis/v1alpha1/image_policy.go

```diff
@@ -204,6 +209,22 @@ type ImagePolicyFilter struct {
 	GitHub *v1.ObjectReference `json:"github,omitempty"`
 }
 
+// ContainerRegistry will define how to fetch images from a container registry.
+// Docker Hub is the only one supported at the moment.
+type ContainerRegistry struct {
+	Name string `json:"name"`
```

Comment:
Should the `ImagePullSecrets` live in here also? Since that's tightly coupled to the registry.

Reply:
Moved inside.